library/build.gradle

```diff
@@ -16,6 +16,10 @@ android {
         targetCompatibility JavaVersion.VERSION_1_6
     }
 
+    lintOptions {
+        abortOnError false
```

Comment:
@hotchemi It was complaining earlier from a broken build. I had to set it as a workaround. It slid through by mistake. Removed.